js/script.js

```diff
@@ -29,6 +32,8 @@ import {
 
 let tasks = [];
 
+let filterBy = "all";
```

Comment:
```suggestion
let filterBy = ALL;
```
use constant variable for filter state and import them from `const.js`.
Like `import { ALL, COMPLETE, INCOMPLETE } from "./const.js"`